android/app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,15 @@
     package="com.ee2_mobile_app">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <!-- Required -->
+    <uses-permission android:name="android.permission.CAMERA" />
+
+    <!-- Include this only if you are planning to use the camera roll -->
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
Remove this

Reply:
ok

Reply:
Consider removing these permission too. QR code scanning may not use camera roll permissions.